deploy/fledged-deployment-swagger.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: fledged
+  namespace: kube-fledged
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: fledged
+  template:
+    metadata:
+      labels:
+        app: fledged
+    spec:
+      containers:
+      - env:
+        - name: KUBERNETES_SERVICE_HOST
+          value: 127.0.0.1
```

Comment:
I suggest to use the existing fledged-deployment.yaml file itself. Just replace the values for KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT using "sed" command before deploying. Maintaing two different deployment yaml files will add complexity in the long run..